client/src/ui/MouseEvent.res

```diff
@@ -0,0 +1,31 @@
+// TODO: can this be replaced with a type from a library?
+@ppx.deriving(show({with_path: false}))
+type rec t = {
+  mePos: VPos.t,
```

Comment:
It took me a few seconds to realize that `mePos` meant `mouseEventPos`.
I'd like to recommend `pos` or `mousePos` as alternatives.

Reply:
Agreed, but I don't wanna make changes to types I just move around